weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage.tsx

```diff
@@ -405,8 +407,22 @@ const CallTraceView: React.FC<{call: WFCall; treeOnly?: boolean}> = ({
     [callClass]
   );
 
+  // Scroll selected call into view
+  useEffect(() => {
+    // The setTimeout here is a hack; without it scrollToIndexes will throw an error
+    // because virtualScrollerRef.current inside the grid is undefined.
+    // See https://github.com/mui/mui-x/issues/6411#issuecomment-1271556519
+    setTimeout(() => {
+      const rowIndex = apiRef.current.getRowIndexRelativeToVisibleRows(
+        call.callID()
+      );
+      apiRef.current.scrollToIndexes({rowIndex});
+    }, 0);
```

Comment:
even though it is effectively an immediate, it still might be worth returning a timeout invalidation